src/ui/app.d.ts

```diff
@@ -0,0 +1,13 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+// and what to do when importing types
+declare namespace App {
+	// interface Locals {}
```

Comment:
I have a lot of things I want to read up on for svelte-kit but does declaring this namespace do anything if we're commenting out all the default interfaces?

Reply:
This is funky boilerplate from them, the way I read this is it's just a cheap way to avoid the type system getting angry with special svelte things.  Note typescript always just transpiles to javescript without types anyway.